app/src/main/java/ca/llamabagel/transpo/search/data/SearchFilterState.kt

```diff
@@ -0,0 +1,3 @@
+package ca.llamabagel.transpo.search.data
+
+data class SearchFilterState(var stops: Boolean = true, var routes: Boolean = true, var places: Boolean = true)
```

Comment:
It's recommended that data classes contain only immutable `val`s instead of `var`s. If you need to change only one property, use the `copy` method along with named parameters.
```suggestion
data class SearchFilter(val stops: Boolean = true, val routes: Boolean = true, val places: Boolean = true)
```

Also, the State suffix might not be necessary. It's already implied that the data class is a stage object, maybe?